src/Auth/ResetsPasswords.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Corcel\Auth;
+
+use Corcel\Password\PasswordService;
+use Auth;
+
+trait ResetsPasswords {
+	/**
+     * Reset the given user's password.
+     *
+     * @param  \Illuminate\Contracts\Auth\CanResetPassword  $user
+     * @param  string  $password
+     * @return void
+     */
+    protected function resetPassword($user, $password)
+    {
+    	$passwordService = new PasswordService;
+
+        $user->user_pass = $passwordService->wp_hash_password($password);
+
+        $user->save();
+        
+        Auth::guard($this->getGuard())->login($user);
```

Comment:
Here you must use the `Auth` instance from Laravel, but if the user is not using Laravel? I think maybe here it's better to use composition, making this class independent of Laravel `Auth`. What do you think?